insurance/README.md

```diff
@@ -47,15 +46,6 @@
   - Datasets: 'insurance-training-data' with version v2. The base dataset is sourced from AWS
   - Model: 'insurance-model' with version v2
 
-
-### Inference
```

Comment:
Inference not required for monitoring demo

Reply:
We are planning to remove insurance in sklearn branch. We should keep it here